gammapy/modeling/fit.py

```diff
@@ -721,6 +725,75 @@ def covariance_result(self):
         """Optimize result."""
         return self._covariance_result
 
+    def to_dict(self, full_output=False, overwrite_templates=False):
+        """Convert to dictionary."""
```

Comment:
Should we describe here the arguments into the docstring?

Reply:
For both this and the `to_yaml` I did not include them because it not done in other instances of the code for these functions